source/utilities.tex

```diff
@@ -14188,36 +14188,40 @@
 \indexlibrarymember{ec}{to_chars_result}
 @
   // \ref{charconv.to.chars}, primitive numerical output conversion
-  struct to_chars_result {
+  struct to_chars_result {                                              // freestanding
     char* ptr;
     errc ec;
     friend bool operator==(const to_chars_result&, const to_chars_result&) = default;
   };
 
-  constexpr to_chars_result to_chars(char* first, char* last, @\placeholder{integer-type}@ value, int base = 10);
-  to_chars_result to_chars(char* first, char* last, bool value, int base = 10) = delete;
+  constexpr to_chars_result to_chars(char* first, char* last,
+                                     @\placeholder{integer-type}@ value, int base = 10);   // freestanding
+  to_chars_result to_chars(char* first, char* last,
+                           bool value, int base = 10) = delete;         // freestanding
 
-  to_chars_result to_chars(char* first, char* last, @\placeholder{floating-point-type}@ value);
-  to_chars_result to_chars(char* first, char* last, @\placeholder{floating-point-type}@ value, chars_format fmt);
+  to_chars_result to_chars(char* first, char* last,
+                           @\placeholder{floating-point-type}@ value);            // freestanding-deleted
   to_chars_result to_chars(char* first, char* last, @\placeholder{floating-point-type}@ value,
-                           chars_format fmt, int precision);
+                           chars_format fmt);                           // freestanding-deleted
+  to_chars_result to_chars(char* first, char* last, @\placeholder{floating-point-type}@ value,
+                           chars_format fmt, int precision);            // freestanding-deleted
 @%
 \indexlibraryglobal{from_chars_result}%
 \indexlibrarymember{ptr}{from_chars_result}%
 \indexlibrarymember{ec}{from_chars_result}
 @
   // \ref{charconv.from.chars}, primitive numerical input conversion
-  struct from_chars_result {
+  struct from_chars_result {                                            // freestanding
     const char* ptr;
     errc ec;
     friend bool operator==(const from_chars_result&, const from_chars_result&) = default;
   };
 
   constexpr from_chars_result from_chars(const char* first, const char* last,
-                                         @\placeholder{integer-type}@& value, int base = 10);
+                                         @\placeholder{integer-type}@& value, int base = 10);     // freestanding
 
   from_chars_result from_chars(const char* first, const char* last, @\placeholder{floating-point-type}@& value,
-                               chars_format fmt = chars_format::general);
+                               chars_format fmt = chars_format::general);   // freestanding-deleted
```

Comment:
I think it's preferable to have these comments
be on the same line as the declaration's identifier they apply to.

Reply:
I think I've fixed this.